src/app/components/auth/login/login.component.html

```diff
@@ -1,12 +1,61 @@
-<div class="login-container">
-  <app-input [isRequired]="true" [label]="'Username'" [type]="'text'" [theme]="'dark'" [icon]="'assets/images/username_dark.png'"  #formlogin></app-input>
-  <app-input [isRequired]="true" [label]="'Password'" [type]="'password'" [theme]="'dark'" [icon]="'assets/images/password_dark.png'"  #formlogin></app-input>
-  <div class="auth-forgot-div">
-    <span class="btn btn-fire" (click)="formValidate('formlogin')">LOGIN</span>
-    <span class="auth-forgot">Forgot password or username?</span>
-  </div>
-  <div class="auth-signup-row">
-    <span>Start with a new account </span>
-    <a [routerLink]="['/auth/signup']" >Sign Up</a>
-  </div>
+<div class="col s12 m6 offset-m3 left">
+  <ul class="left">
+    <li class="ev-horiz-list"><a routerLink="/auth/login" class="dark-link" routerLinkActive="active-auth"
+                                 (click)="authService.resetForm()">Log In</a></li>
+    <li class="ev-horiz-list"><a routerLink="/auth/signup" class="light-link" routerLinkActive="active-auth"
+                                 (click)="authService.resetForm()">Sign Up</a></li>
+  </ul>
+</div>
+<div class="col s12 m6 offset-m3">
+  <!-- login form -->
+  <form name="loginForm" #loginForm='ngForm' (ngSubmit)="userLogin(loginForm.valid)" id="log-in" novalidate>
+    <!-- username -->
+    <div class="input-field align-left">
+      <input type="text" id="name" class="dark-autofill" name="name" (focusin)="isnameFocused = true"
```

Comment:
@sanketbansal I don't see you using the reusable component for inputs

Reply:
yes, because I am using angulars template-driven form.

Reply:
but then I think there's no use of `reusable components` if we are not using them.

cc: @RishabhJain2018 @lunayach

Reply:
Thanks @Sanji515 for pointing out this. @sanketbansal can you look at the `input component` in `utility` folder. The component seems to be well generalized. Also, it's `validateInput( )` takes care of things like validating e-mail etc., so we'll not require `email.validator.ts` as well.

Reply:
Yeah but it requires a lot of restructuring of the code both in app-input component and auth components. I will create a **seperate PR** for that so that i can use template driven from using app-input component.
Also, I don't think there is any functionality to compare passwords which might need to be added in that.

Reply:
Also, There is a difference between validator directive and validateInput() method significantly.
As validator direvtive checks errors at every interaction of user with the input element.

Reply:
Yeah, but that can be taken care while using appropriate events, right?
Still, I feel that the same pattern should be followed in the code. So, if something similar has already been implemented, we should definitely try to use that.
And we can do this re-structuring in a separate PR. @RishabhJain2018, so, should we open an issue for it?

Reply:
> Yeah, but that can be taken care while using appropriate events, right?

yeah, but I think its better and effective to use angular features instead of making our own events for that.

Reply:
As, validator directive events are being handled by angular internally.